src/controllers/productController.js

```diff
@@ -11,6 +11,24 @@ const getProducts = catchAsync(async (req, res) => {
     return res.status(200).json({ message : products })
 })
 
-module.exports = {
-    getProducts
-}
\ No newline at end of file
+    
+const getProductDetail = catchAsync(async (req, res) => {
+
+    const userId = req.userId;
+
+    const productId = req.params.productId;
+
+    const productInfo = await productService.getProductDetail(userId, productId);
+
+
+    return res.status(200).json({ message : productInfo });
+    
+})
+
+
+
+
+  module.exports = {
+    getProducts,
+    getProductDetail
+}
```

Comment:
가독성을 위한 줄 바꿈이 필요하다면 한 줄로 충분합니다! 또한 변수 선언 뒤 할당하는 부분은 다음을 참조 하시면 좋을 것 같습니다. 다만 이러한 컨벤션 적인 부분은 일반적으로 사용되는 것이지 무조건적인 정답은 아니다 라는 것도 기억해주세요! 

해당 코멘트 또한 추후에 꼭 반영 될 수 있게 해주세요~
```suggestion
    
const getProductDetail = catchAsync(async (req, res) => {
    const userId = req.userId;
    const productId = req.params.productId;

    const productInfo = await productService.getProductDetail(userId, productId);

    return res.status(200).json({ message : productInfo });
})

module.exports = {
    getProducts,
    getProductDetail
}
```

Reply:
수정했습니다.